### PR TITLE
doc/reference: improve description of storage  option

### PR DESCRIPTION
**Pull request by simondeziel**
I'm not familiar with LVM so please let me know if `LVM pool` is wrong or confusing.

### CODE REVIEW

File: doc/reference/storage_lvm.md [hunk 1]
```diff
@@ -42,14 +42,14 @@ The following configuration options are available for storage pools that use the
...
-`rsync.bwlimit`               | string                        | `0` (no limit)                            | The upper limit to be placed on the socket I/O when `rsync` must be used to transfer storage entities
+`rsync.bwlimit`               | string                        | `0` (no limit)                          | The upper limit to be placed on the socket I/O when `rsync` must be used to transfer storage entities
 `rsync.compression`           | bool                          | `true`                                  | Whether to use compression while migrating storage pools
 `size`                        | string                        | auto (20% of free disk space, >= 5 GiB and <= 30 GiB) | Size of the storage pool when creating loop-based pools (in bytes, suffixes supported)
-`source`                      | string                        | -                                       | Path to block device or loop file or file system entry
+`source`                      | string                        | -                                       | Path to an existing block device, loop file or LVM pool
```
Comment (tomponline) [inline doc/reference/storage_lvm.md:52]: LVM volume group

File: doc/reference/storage_dir.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Key                           | Type                          | Default
 :--                           | :---                          | :------                                 | :----------
 `rsync.bwlimit`               | string                        | `0` (no limit)                          | The upper limit to be placed on the socket I/O when `rsync` must be used to transfer storage entities
 `rsync.compression`           | bool                          | `true`                                  | Whether to use compression while migrating storage pools
-`source`                      | string                        | -                                       | Path to block device or loop file or file system entry
+`source`                      | string                        | -                                       | Path to an existing block device, loop file or file system entry
```
Comment (tomponline) [inline doc/reference/storage_dir.md:29]: I need to check if it will accept a block device.

Comment (tomponline) [reply doc/reference/storage_dir.md:29]: @simondeziel it does not. It should be a path to a directory only.

Comment (simondeziel) [reply doc/reference/storage_dir.md:29]: Thanks for checking, PR #10749

**Pull request merged**